addons/storyshots/stories/directly_required/Button.js

```diff
@@ -1,28 +0,0 @@
-import PropTypes from 'prop-types';
```

Comment:
There's no difference in requiring it from a local file or from a npm package?

I'd like to remove duplication..